deploy/olm-catalog/ocs-operator/ocs-operator.package.yaml

```diff
@@ -1,4 +1,6 @@
 packageName: ocs-operator
 channels:
-- name: alpha
-  currentCSV: ocs-operator.v0.0.2
+- name: 4.2.0
+  currenCSV: ocs-operator.v0.0.1
```

Comment:
Should this be `v0.0.2`?

Reply:
The 0.0.2 CSV no longer exists with this change. And release-4.2 has the CSV version set to 0.0.1.